File: testprojects/scala_main/subgraphs/test2sg/code/src/main/scala/org/scala/main/job1/graph/test2sg_1/RowDistributor_1_1.scala
package org.scala.main.job1.graph.test2sg_1

import io.prophecy.libs._
import com.scala.main.job1.config.ConfigStore._
import com.scala.main.job1.udfs.UDFs._
import com.scala.main.job1.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import java.time._

object RowDistributor_1_1 {

  def apply(spark: SparkSession, in: DataFrame): (DataFrame, DataFrame) =
    (in.filter(col("`c- short`") > lit(-10)),
     in.filter(col("`c  - int`") > lit(-100))
    )

}

File: testprojects/master_python_mainone/pipelines/StreamingWorking/code/streamingpipelinemain/graph/all_components_main_1/RowDistributor_1_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from streamingpipelinemain.config.ConfigStore import *
from streamingpipelinemain.udfs.UDFs import *

def RowDistributor_1_1(spark: SparkSession, in0: DataFrame) -> (DataFrame, DataFrame):
    df1 = in0.filter(((col("`c- short`") > lit(-1)) & (col("p_short") > lit(1))))
    df2 = in0.filter((col("`c- short`") > lit(5)))

    return df1, df2
